sashimi/hardware/cameras/__init__.py

```diff
@@ -0,0 +1,180 @@
+from sashimi.config import read_config
+import time
+import numpy as np
+
+conf = read_config()
+
+
+class BasicCamera:
+    def __init__(self):
+        super().__init__()
+        self.encoding = "utf-8"
+        self.camera_id = conf["camera"]["id"]
+        self.image_height = 2048
```

Comment:
it is inconsistent to have image split into height and weight and the frame shape as a tuple. I would instead suggest having a property `sensor_resolution` that is a tuple